projects/schematics/src/migrations/3_0/methods-and-properties-deprecations/data/store-finder-group.actions.migration.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  FIND_STORES_CLASS,
+  SEARCH_CONFIG,
+  SPARTACUS_CORE,
+  StoreFinderActions,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const STORE_FINDER_GROUP_ACTIONS_MIGRATION: MethodPropertyDeprecation[] = [
```

Comment:
I don't know how crucial it is, but there seems to be a convention of adding the path of the file in a comment above here in other migration files.

Reply:
👍 

For this case, I pushed the path to the index because of how it was designed in the previous deprecation in `2.0` for `cms-group-actions.migration.ts` as I believe this specific migration file will support 'x amount' of actions for this group set

Reply:
What do you think would be best? 
1) the path to the index
or
2) the path to the file, but we would have 'x amount' of comment for path to file as this migration file supports other deprecations from a store finder action if exist

Reply:
I think what you did is fine.